src/puppetlabs/puppetdb/import.clj

```diff
@@ -40,7 +40,7 @@
       (do (log/infof "Importing report from archive entry '%s'" path)
           (command-fn :store-report
                       (:store_report command-versions)
-                      (reports/sanitize-report (utils/read-json-content tar-reader true))))
```

Comment:
It wasn't used which is ones of the reasons I removed it, another reason it that the only real thing that needs/uses this sanitization appears to be benchmarking, which is still using v5, so I figured I should probably handle that in a different patch, and why upgrade it here when it's not used.